sdk/deviceupdate/azure-iot-deviceupdate/README.md

```diff
@@ -54,17 +54,6 @@ All Device Update for IoT Hub service operations will throw a ErrorResponseExcep
 
 For example, if you use the `getUpdateWithResponse` operation and the model you are looking for doesn't exist, you can catch that specific HttpStatusCode to decide the operation that follows in that case.
 
-``` java com.azure.iot.deviceupdate.DeviceUpdateAsyncClient.notfound
```

Comment:
Why delete those codesnippet?
This document shows how to add codesnippet: https://github.com/Azure/azure-sdk-tools/blob/main/packages/java-packages/codesnippet-maven-plugin/README.md

Reply:
SDK would prefer we had as least one sample in README.md, so customer know how to authenticate and call API.

Please see above doc on how to sync your sample java to the README.md

Alternative doc https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods-Quickstart-with-AutoRest#samples

Reply:
Sorry, misunderstood the purpose of those. I brought them back. Hope it helps.